app.js

```diff
@@ -0,0 +1,60 @@
+// empty array to store the books
+let books = [];
+
+//  function to add a new book to the collection, with title and author
+function addBook(title, author) {
+  const book = {
+    title,
+    author,
+  };
+  books.push(book);
+}
+
+// function to display the books in the collection
+function displayBooks() {
+  const tableBody = document.querySelector('#book-table tbody');
+  tableBody.innerHTML = '';
+  books.forEach((book) => {
+    const row = tableBody.insertRow();
+    const titleCell = row.insertCell();
+    const authorCell = row.insertCell();
+    const removeCell = row.insertCell();
+    titleCell.innerText = book.title;
+    authorCell.innerText = book.author;
+    removeCell.innerHTML = `<button onclick="removeBook('${book.title}')">Remove</button>`;
+  });
+}
+
+// function to remove a book from the collection
+function removeBook(title) {
+  books = books.filter((book) => book.title !== title);
+  displayBooks();
+}
```

Comment:
- [ ] Because you're not setting IDs on books and here you're removing a book by its title which is not unique, this causes a book of the same title to be removed. Remember, we can have books of the same title but with a different author 👍🏽
- [ ] What if you could remove a book like this:
```js
const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```